wire-ios-data-model/Tests/MLS/MockMLSActionExecutor.swift

```diff
@@ -17,15 +17,26 @@
 //
 
 import Foundation
-@testable import WireDataModel
 import WireCoreCrypto
 import Combine
 
-class MockMLSActionExecutor: MLSActionExecutorProtocol {
+@testable import WireDataModel
+
+final class MockMLSActionExecutor: MLSActionExecutorProtocol {
```

Comment:
`MLSActionExecutor` is an actor, however, I didn't manage to make `MockMLSActionExecutor` an actor too, because of the method `onEpochChanged()` which cannot be isolated (no async in the protocol).

I added a `serialQueue` so that the mock properties can be set from different threads.